android/build.gradle

```diff
@@ -51,7 +46,7 @@ dependencies {
   implementation "com.google.zxing:core:3.3.0"
   implementation "com.drewnoakes:metadata-extractor:2.9.1"
   implementation "com.google.android.gms:play-services-vision:${safeExtGet('googlePlayServicesVersion', '15.0.2')}"
-  implementation "com.android.support:exifinterface:${safeExtGet('supportLibVersion', '27.1.0')}"
-  implementation "com.android.support:support-annotations:${safeExtGet('supportLibVersion', '27.1.0')}"
-  implementation "com.android.support:support-v4:${safeExtGet('supportLibVersion', '27.1.0')}"
+  implementation "androidx.exifinterface:exifinterface:1.0.0-beta01"
```

Comment:
why is this beta? is it not stable yet?

Reply:
Good catch. Thats what the Android Studio migration tool migrated the dependency to. Looking at [this](https://developer.android.com/jetpack/androidx/migrate), it looks like the stable release versions are out. I'll update with another commit.